en/docs/graphql-api.md

```diff
@@ -0,0 +1,79 @@
+
+
+## Develop
+
+To create a GraphQL you need to add a GraphQL component.
+
+Choreo comes with the Web Editor which is an IDE designed based on Visual Studio Code for developers to implement API functionality. At present, developers can design GraphQL APIs in the Web Editor by writing code, and test the functionality via cURL commands before committing the design to a private/Choreo-managed GitHub repository. Once you commit the design, the GraphQL API is available in the Choreo Console where you can deploy, test, and observe it. 
+
+## Lifecycle
+
+Once you implement a GraphQL API, it needs to be built, tested, and eventually put into production. Choreo executes a CI/CD pipeline to manage the API lifecycle. Choreo stores the source code of APIs in a GitHub repository for the user account. This GitHub repository can be a Choreo-managed repository or a GitHub repository that you own.  It also provides the capability for developers to connect their own GitHub repositories containing the source code of their APIs. The CI/CD pipeline in Choreo takes an API through the complete process, starting from code checkout to compile, build, test, and finally deploy.
+
+## Deploy
+
+The following diagram illustrates the procedure to deploy an API in Choreo to the default development environment.
+
+![Deploy API](assets/img/rest-apis/deploy-api.png){.cInlineImage-full}
+
+Choreo runs a professional, enterprise-grade CI/CD process to deploy APIs to their runtime(data plane) clusters. The data plane of Choreo runs on a Kubernetes stack under the hood. Therefore, you can benefit from its features such as auto-scaling, auto-healing, secret-management, liveness and readiness checks, etc.
+
+Once you deploy a GraphQL API to the development environment, it gets exposed through an API Gateway with API security enabled. At this time, Choreo provides a test URL for the API, which you can use to verify the functionality of the API.
+
+## Choreo Environments
+
+Choreo provides a development environment by default and a production environment in two separate Kubernetes clusters. Once you deploy an API to the development environment and verify its functionality, you can promote the API to the production environment.
+
+![Choreo environments](assets/img/graphql/choreo-environments.png){.cInlineImage-full}
+
+Once you promote an API to production, Choreo provides a second URL that you can use in a production application to invoke the API.
+
+## Test
+
+Once you deploy your GraphQL API, you can test it using the GraphQL Console. The GraphQL Console is an interactive UI that guides you to write queries and mutations to test the GraphQL API.
+
+Choreo by default uses OAuth2.0 authentication to secure GraphQL APIs. Therefore, the GraphQl Console allows you to generate test keys to test the APIs.
+
+![GraphQL Console](assets/img/graphql/graphql-console.png){.cInlineImage-full}
```

Comment:
In this image, let's only show the explorer and docs views. We can hide the history as it is not that important for the initial testing. 

Example: 
![image](https://user-images.githubusercontent.com/8376040/185425138-23bd8283-7463-4703-9b43-beba9ffb41d7.png)

Reply:
Done. Updated the image.